components/StudentCard.tsx

```diff
@@ -0,0 +1,19 @@
+import { Student } from "../models/users";
+import styles from "../styles/Components.module.css";
+
+type StudentCardProps = { student: Student };
```

Comment:
Lets only take in the things we need to render this instead of the entire object